rules/windows/credential_access_lsass_openprocess_api.toml

```diff
@@ -157,6 +159,7 @@ api where host.os.type == "windows" and
         "?:\\Windows\\System32\\RtkAudUService64.exe",
         "?:\\Windows\\System32\\wbem\\WmiPrvSE.exe",
         "?:\\Windows\\SysWOW64\\wbem\\WmiPrvSE.exe"
+    ) and not ?process.code_signature.trusted == false
```

Comment:
Will this work? IINM, I believe the comparison defaults to `null` if the optional field is missing, making this logic essentially 

```sql
and not null
```

which means, it will never trigger when missing, which is always for MDE.


```suggestion
    ) and (?process.code_signature.trusted == null or ?process.code_signature.trusted == false)
```

This changes it to say:
* _if it is missing (MDE) then proceed to match) else_
* _if it **is** present, match if false_

Reply:
Yeah, it works
![image](https://github.com/elastic/detection-rules/assets/26856693/2f85aae9-c468-4f3d-9469-390886b4aba2)

Reply:
yep, disregard, the _field_ not the _comparison_ defaults to `null`, so it works as expected 👍